src/screens/TopicPageScreen/index.js

```diff
@@ -352,7 +356,7 @@ const TopicPageScreen = (props) => {
       const dy = y - lastDragY;
       if (y <= 30) {
         showHeaderAnimation();
-      } else if (dy - 20 > 0) {
+      } else if (dy - 20 > dimen.size.TOPIC_CURRENT_ITEM_HEIGHT / 2) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the condition for triggering the header animation seems fine as long as the `TOPIC_CURRENT_ITEM_HEIGHT` is always guaranteed to be a number. If there's a chance it could be `undefined` or `null`, this could cause a runtime error. You should add a check to ensure `TOPIC_CURRENT_ITEM_HEIGHT` is a number before using it in a calculation.
```diff
- } else if (dy - 20 > dimen.size.TOPIC_CURRENT_ITEM_HEIGHT / 2) {
+ } else if (dimen.size.TOPIC_CURRENT_ITEM_HEIGHT && dy - 20 > dimen.size.TOPIC_CURRENT_ITEM_HEIGHT / 2) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->